internal/venice-common/src/main/java/com/linkedin/venice/pubsub/api/PubSubMessage.java

```diff
@@ -43,4 +43,6 @@ default int getPartition() {
    * @return whether this message marks the end of bootstrap.
    */
   boolean isEndOfBootstrap();
+
+  PubSubMessageHeaders getPubSubMessageHeaders();
```

Comment:
you can have a default implementation of returning empty headers

Reply:
Added this change. Thanks.